### PR TITLE
CLOUDP-265398 Add new test to manage containers created outside of cli

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Add new test to manage containers created outside of cli

_Jira ticket:_ CLOUDP-265398

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/container/docker_test.go [hunk 1]
```diff
@@ -94,3 +97,86 @@ func TestPortMappingFlag(t *testing.T) {
...
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			result, err := parsePortMapping(tt.input)
+			if err != nil {
+				t.Error(err)
+			}
+			if !reflect.DeepEqual(result, tt.expected) {
+				t.Errorf("parsePortMapping(%s) = %+v; want %+v", tt.input, result, tt.expected)
+			}
```
Comment (gssbzn) [inline internal/container/docker_test.go:179]:
```suggestion
			if diff := deep.Equal(tt.expected, result); diff != nil {
				t.Fatalf("diff: %v", diff)
			}
```

File: internal/container/docker_test.go [hunk 2]
```diff
@@ -94,3 +97,86 @@ func TestPortMappingFlag(t *testing.T) {
...
+				ContainerPort: 8080,
+			}},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			result, err := parsePortMapping(tt.input)
+			if err != nil {
+				t.Error(err)
```
Comment (gssbzn) [inline internal/container/docker_test.go:175]:
```suggestion
				t.Fatal(err)
```
or the next check will run anyway

File: internal/container/docker_test.go [hunk 3]
```diff
@@ -94,3 +97,86 @@ func TestPortMappingFlag(t *testing.T) {
...
+			input: "localhost->8080",
+			expected: []PortMapping{{
+				HostAddress:   "localhost",
+				ContainerPort: 8080,
+			}},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```
Comment (gssbzn) [inline internal/container/docker_test.go:172]:
```suggestion
		t.Run(tt.name, func(t *testing.T) {
			t.Parallel()
```

**Review by jeroenvervaeke — DISMISSED** [review]

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**